docs/examples/bitops+terraform+ansible-blog/README.md

```diff
@@ -5,12 +5,12 @@ Welcome to Bitops! This serves as a starting point for deploying your applicatio
 This repo can be run as is with
 ```
 docker run \
--e ENVIRONMENT="test" \
+-e BITOPS_ENVIRONMENT="test" \
 -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
 -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
 -e AWS_DEFAULT_REGION="us-east-2" \
 -v $(pwd):/opt/bitops_deployment \
 bitovi/bitops:latest
 ```
 
-For more information, check out official BitOps docs https://bitovi.github.io/bitops/
\ No newline at end of file
+For more information, check out official BitOps docs https://bitovi.github.io/bitops/
```

Comment:
The only note here, it would be nice to cross-link to the original blog post URL